data/json/items/armor.json

```diff
@@ -5983,6 +5983,29 @@
         "bashing" : -5,
         "coverage" : 50,
         "material_thickness" : 1
+    },   
+    {   "type" : "ARMOR",
+        "id" : "hat_sombrero",
+        "name" : "sombrero",
+        "weight" : 140,
+        "color" : "orange",
+        "covers" : ["HEAD"],
+        "to_hit" : 0,
+        "symbol" : "[",
+        "description" : "A fancy orange sombrero with traditional white and yellow hand stitching.  The wide brim keeps the sun out of your eyes, and a thin strap hooks around your chin.",
+        "price" : 5500,
+        "material" : ["cotton", "null"],
+        "volume" : 14,
+        "cutting" : 0,
+        "warmth" : 1,
+        "phase" : "solid",
+        "environmental_protection" : 4,
+        "encumbrance" : 20,
+        "//" : "Head encumbrance stops you from wearing additional headgear.  No helmet over a sombrero, sorry.  But brimmed gear gets the anti-glare flag!",
+        "flags" : ["FANCY", "BELTED", "SUN_GLASSES"],
```

Comment:
K, I'll set that.  It seems a bit odd in game to see "this gear is generally worn over clothing" with the OUTER flag but no big either way.